spec/system/admin/order_cycles/simple_spec.rb

```diff
@@ -531,7 +531,8 @@
         )
 
         # When I save, any exchanges that I can't manage remain
-        click_button 'Save'
+        # overlapping warning, we need to use 'node.trigger("click")'
+        page.find(:button, "Save").trigger("click")
```

Comment:
Can we dismiss the warning first? Otherwise this is a UX issue. Users can't "find button and trigger save" with their mouse.